### dev console

```
window_watchers-1PzqCc51.js:1 Error: A <SelectItem /> must have a value prop that is not an empty string. This is because the Select value can be set to an empty string to clear the selection and show the placeholder.
    at setup (main-Bg7YgpwL.js:3880:17458)
    at Ds (window_watchers-1PzqCc51.js:1:25548)
    at jp (window_watchers-1PzqCc51.js:3:44383)
    at va (window_watchers-1PzqCc51.js:3:44172)
    at W (window_watchers-1PzqCc51.js:3:27391)
    at F (window_watchers-1PzqCc51.js:3:27284)
    at _ (window_watchers-1PzqCc51.js:3:24002)
    at w (window_watchers-1PzqCc51.js:3:25598)
    at T (window_watchers-1PzqCc51.js:3:26990)
    at _ (window_watchers-1PzqCc51.js:3:23938)
```

--- a/src/components/tabs/LedStripTab.vue
+++ b/src/components/tabs/LedStripTab.vue
@@ -393,7 +393,7 @@ const gridLeds = reactive(
 
 // Selection state
 const selectedIndices = ref(new Set());
-const selectedFunction = ref("");
+const selectedFunction = ref("none");
 const selectedColorIndex = ref(0);
 
 // Modifier states
@@ -448,7 +448,7 @@ const showSpecialColors = computed(() => {
 
 // USelect item arrays
 const functionItems = computed(() => [
-    { label: t("ledStripFunctionNoneOption"), value: "" },
+    { label: t("ledStripFunctionNoneOption"), value: "none" },
     { label: t("ledStripFunctionColorOption"), value: "function-c" },
     { label: t("ledStripFunctionModesOption"), value: "function-f" },
     { label: t("ledStripFunctionArmOption"), value: "function-a" },
@@ -628,7 +628,7 @@ function handleSelectionComplete() {
 
     // Update selected function
     const baseFunc = led.functions.find((f) => baseFuncs.includes(f));
-    selectedFunction.value = baseFunc ? `function-${baseFunc}` : "";
+    selectedFunction.value = baseFunc ? `function-${baseFunc}` : "none";
 
     // Update directions
     activeDirections.value = new Set(led.directions);
@@ -668,7 +668,7 @@ function clearSelected() {
     });
 
     // Clear UI state
-    selectedFunction.value = "";
+    selectedFunction.value = "none";
     activeDirections.value.clear();
     Object.keys(overlayStates).forEach((key) => {
         overlayStates[key] = false;
@@ -689,7 +689,7 @@ function clearAll() {
     });
 
     // Clear UI state
-    selectedFunction.value = "";
+    selectedFunction.value = "none";
     activeDirections.value.clear();
     Object.keys(overlayStates).forEach((key) => {
         overlayStates[key] = false;

